### PR TITLE
Add name tejaswini to participant.md

### PR DESCRIPTION
**Pull request by universet**

### CODE REVIEW

**Comment by radhey1851** [comment]
Looks good to me(LGTM), will merge

File: participants.md [hunk 1]
```diff
@@ -5,3 +5,5 @@ Classes on JS, React, redux and other tools for Front end development.
 ### Participants
 
 Trainer: Radheshyam Etikala
+
+Tejaswini Jadhav
```
Comment (radhey1851) [inline participants.md]: Maybe you can add a '-' here so that it shows as a list

**Review by radhey1851 — COMMENTED** [review]
pls look at my comments

File: participants.md (hunk 1 above)
Comment (universet) [reply participants.md]: Updated!

**Pull request merged**